client/components/marketing-survey/cancel-jetpack-form/index.tsx

```diff
@@ -65,6 +71,38 @@ const CancelJetpackForm: React.FC< Props > = ( {
 		isSiteAutomatedTransfer( state, purchase.siteId )
 	);
 
+	const cancellationOffer = useSelector( ( state ) => {
+		const offers = getCancellationOffers( state, purchase.id );
+		if ( 1 === offers.length ) {
+			return offers[ 0 ];
+		}
+
+		return null;
+	} );
+
+	const fetchingCancellationOffers = useSelector( ( state ) =>
+		isFetchingCancellationOffers( state )
```

Comment:
I'm seeing you added the purchase id in #66051.

Reply:
Yep, found that while fixing some previous feedback :)